pkg/cip/cli/run.go

```diff
@@ -159,8 +159,9 @@ func RunPromoteCmd(opts *RunOptions) error {
 		sc, err = reg.MakeSyncContext(
 			mfests,
 			opts.Threads,
-			opts.DryRun,
-			opts.UseServiceAcct)
+			opts.NoDryRun,
```

Comment:
I’m not sure what does that function expects, but if it takes dryRun, shouldn’t we also invert it here?

For example, if takes dryRun and it is dryRun, noDryRun would be set to false, so this function would run as noDryRun instead of dryRun.

Reply:
@xmudrii -- It's inverted below, starting with https://github.com/kubernetes/release/pull/1806/files#diff-94fc64f6c6f758dcf8d71f20a75b9e43e7093ec5eb51c2ca80ad45532a31b423R62.